web/js/main.js

```diff
@@ -60,6 +60,11 @@ window.onload = () => {
   loadingIndicator.delayed(promise, 1000);
   promise
     .done((config) => {
+      // Perform check to see if app was in the midst of a tour
+      if (parameters.tr) {
+        // Gets the extent of the first step of specified tour and overrides current view params
+        parameters = util.fromQueryString(config.stories[parameters.tr].steps[0].stepLink);
```

Comment:
I get a little nervous when I see object/array access this many levels deep without any `undefined` checking.  I don't _think_ there would be a case where the first step of a tour doesn't have a `steplink` so it should be OK but... may be worth using [lodash's `get` function](https://lodash.com/docs/4.17.15#get) in the future to access nested arrary/object properties like this safely.